tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchResponseBuilderFacts.cs

```diff
@@ -357,6 +360,145 @@ public void UsesVersionSpecificDownloadCount()
                 Assert.Equal(4, response.Data[0].DownloadCount);
             }
 
+            [Fact]
+            public void SortDownloadsAscUsingAuxilaryFile()
```

Comment:
These tests a little bit long because of the data setup. I would recommend creating a new field, like `_fiveSearchResults`, that is created in the constructor similar to how `_searchResult` and `_manySearchResults` are created. 

I would also recommend building each result by using `Data.SearchDocument` and then modifying its fields. Note that `Data.SearchDocument` creates a new document each time it is accessed. For example:

```cs
var document1 = Data.SearchDocument;
document1.PackageId = "Hello world";
document1.TotalDownloadCount = 123
```